op-mode-definitions/date.xml.in

```diff
@@ -37,28 +37,6 @@
         </properties>
         <command>/bin/date "$3"</command>
       </tagNode>
-      <node name="date">
```

Comment:
I wonder if a command to force time update by hand can still be useful. Since ntpdate is in a separate package and doesn't depend on the NTPd package, I'm not sure if there's any harm in keeping it.

I only think we need to dehardcode the NTP server name from its script.

Or are there reasons why it's better to remove it?

Reply:
Chrony can do this implicitly once started, no need to force the time setup.

This "hack" only worked when ntpd service was not running as both used the same communication socket. The "new" way would be to simply enable NTP server with chrony and everything else works out of the box.